Ports/ffmpeg/package.sh

```diff
@@ -17,6 +17,7 @@ configure() {
         --enable-cross-compile \
         --enable-gpl \
         --enable-libx264 \
+        --enable-libx265 \
```

Comment:
Make sure to also add `x265` to the dependencies.

Reply:
Thanks, added.